src/Common/Internal/Utilities.php

```diff
@@ -368,27 +368,15 @@ public static function rfc1123ToDateTime($date)
     /**
      * Generate ISO 8601 compliant date string in UTC time zone
      *
-     * @param int $timestamp The unix timestamp to convert
-     *     (for DateTime check date_timestamp_get).
+     * @param \DateTimeInterface $date The date value to convert
      *
      * @return string
      */
-    public static function isoDate($timestamp = null)
+    public static function isoDate($date)
```

Comment:
If the parameter only accepts `\DateTimeInterface`, I strongly advice a type is added here for a type check.

Reply:
Also, it looks like a breaking change to me... @XiaoningLiu thoughts?

Reply:
The method type should be added now, thanks! I assumed that the classes under `..\Internal` are not used outside this codebase and did introduce a breaking chance (the function was only used once in `MicrosoftAzure\Storage\Tests\Unit\Table\Models\EntityTest` anyway). If you'd like I will re-add handling for integers/timestamps.

Reply:
The only worry I had is that user could use it without noticing that it is an internal interface. Anyway I'm OK with the current implementation. @XiaoningLiu Please make sure this is noted in the 0.19.0 Changelog.md. Also @z38 thanks for the contribution!